libs/PF_MultiPageEdit.js

```diff
@@ -0,0 +1,337 @@
+/**
```

Comment:
It seems there's nothing referring to `PF_MultiPageEdit.js` in code?

Reply:
If that's the case then perhaps we can just go with stock `REL1_39`.  I'll dig around to see if I can confirm it isn't being used.

Reply:
Removed in https://github.com/Wikia/mediawiki-extensions-PageForms/pull/2/commits/108bc8729848425e7f3b31551613e0a51ac302cb